api/src/test/scala/za/co/absa/loginsvc/rest/config/provider/ConfigProviderTest.scala

```diff
@@ -48,6 +48,7 @@ class ConfigProviderTest extends AnyFlatSpec with Matchers  {
     activeDirectoryLDAPConfig.order shouldBe 2
     activeDirectoryLDAPConfig.serviceAccount.username shouldBe "CN=svc-ldap,OU=Users,OU=CORP Accounts,DC=corp,DC=dsarena,DC=com"
```

Comment:
Let just maybe strip the dsarena.com domain from the tests?

Reply:
Amended